docs/source/visualisation/kedro-viz_visualisation.md

```diff
@@ -66,46 +66,82 @@ By convention, a [pipeline can be defined as having different layers](../resourc
 
 For example, the [data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71) labels datasets according to the stage of the pipeline (e.g. whether the data has been cleaned).
 
-You can add a `layer` attribute to the datasets in the Data Catalog, which is reflected in the Kedro-Viz visualisation.
+As of Kedro version 18.9, we've made a change to the way layers are defined in the Data Catalog. The definition is now included under the kedro-viz metadata, where previously it was an attribute specified within a dataset's definition. 
```

Comment:
I'm wondering if we could rephrase this part as just pretending this has been like that forever, and then adding a section or callout about how to migrate old catalogs. I think the version of Kedro that introduced the feature is not very interesting for many readers (and it becomes less and less interesting the more time it passes, hence it becomes documentation debt). What are your thoughts @stichbury?

Reply:
I'm not sure about gaslighting our users 😆 

I have made a commit to change the way we describe the change based on a conversation on Slack.